testdata/Links, inline style.html

```diff
@@ -8,4 +8,10 @@
 
 <p><a href="/url/" title="title has spaces afterward">URL and title</a>.</p>
 
+<p><a href="-" title="title">hyphen and title</a>.</p>
+
+<p><a href="-" title="title with parentheses()">hyphen and title</a>.</p>
+
+<p><a href="-" title="?=title with other chars(#var)">hyphen and title</a>.</p>
```

Comment:
what happens when you test with a single paren? (i.e. not an open close pair)

Reply:
will most likely fail! I'll check and try to fix tonight

Reply:
fixed; added test cases